fairer-iOS/fairer-iOS/Network/Router/RulesRouter.swift

```diff
@@ -5,4 +5,47 @@
 //  Created by 홍준혁 on 2023/02/22.
 //
 
-import Foundation
+import Moya
+
+enum RulesRouter {
+    case getRules
+    case postRuels(ruleName: String)
```

Comment:
p1;
```suggestion
    case postRules(ruleName: String)
```
오타.. 수정 부탁드립니당...

Reply:
https://github.com/fairer-iOS/fairer-iOS/pull/112/commits/f638deb5ffe8a664ed3e4a62c42ce1e3659e2a33